jenkins-scripts/ign_common-default-devel-windows-amd64.bat

```diff
@@ -4,10 +4,8 @@ set VCS_DIRECTORY=gz-common
 set PLATFORM_TO_BUILD=x86_amd64
 set IGN_CLEAN_WORKSPACE=true
 
-set DEPEN_PKGS=assimp ffmpeg freeimage gts tinyxml2
 for /f %%i in ('python "%SCRIPT_DIR%\tools\detect_cmake_major_version.py" "%WORKSPACE%\%VCS_DIRECTORY%\CMakeLists.txt"') do set GZ_MAJOR_VERSION=%%i
 if %GZ_MAJOR_VERSION% GEQ 5 (

```

Comment:
Can also remove the `if` block then.

Reply:
ouch, removed [05a2dcf](https://github.com/gazebo-tooling/release-tools/pull/907/commits/05a2dcf2bc20885ad41528f6fa69817bd2cfb5a1)